sphinx_gallery/gen_rst.py

```diff
@@ -186,6 +186,8 @@ def extract_intro(filename):
     """ Extract the first paragraph of module-level docstring. max:95 char"""
 
     docstring, _ = get_docstring_and_rest(filename)
```

Comment:
My original idea was (untested so maybe there is a problem with it), just change `get_docstring_and_rest` to return `"", full_content_of_the_file including docstring`. Then you can run later the example normally and you don't have more code changes to do. Maybe we require an non empty docstring in which case just return something like "A SyntaxError happened while parsing the example" as the docstring.